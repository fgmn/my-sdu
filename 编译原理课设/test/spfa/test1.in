10
20
0
7
8
0
9
11
0
8
10
0
2
7
0
6
1
1
4
7
1
7
14
2
9
9
2
9
17
2
6
13
2
5
1
3
8
23
3
5
5
3
6
12
3
5
23
3
5
29
4
9
26
4
9
21
4
5
26
8
9
18
