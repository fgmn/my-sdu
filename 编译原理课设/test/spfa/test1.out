0
590558003
7
590558003
590558003
8
1
8
10
11
